src/website/src/pages/ec/components/forms/select/docs/usage.mdx

```diff
@@ -5,6 +5,8 @@ order: 1
 
 import { Paragraph, Anatomy, Link } from '@ecl/website-components';
 
+# Single-dropdown select
```

Comment:
Below there's ` when there are more than 10 options to select from`
The wiki says `when there are more than 5 options to select from`